src/patternfly/base/_variables.scss

```diff
@@ -232,6 +232,10 @@
   // Icons
   --#{$pf-global}--icon--Color--light: #{$pf-v5-global--icon--Color--light};
   --#{$pf-global}--icon--Color--dark: #{$pf-v5-global--icon--Color--dark};
+  --#{$pf-global}--icon--Color--light--light: #{$pf-v5-global--icon--Color--light--light};
```

Comment:
I would very much appreciate a comment clarifying which variable is which (e.g. --[light/dark component]--[light/dark theme] or whatever it means :-)